kubecost-exporter.md

```diff
@@ -11,9 +11,26 @@ If you would prefer to not use the recommended install option and just deploy th
 
 1. Apply the combined YAML:
 
-    ```
-    kubectl apply -f https://raw.githubusercontent.com/kubecost/cost-model/develop/kubernetes/exporter/exporter.yaml --namespace cost-model
-    ```
+    1.a.
+
+      ```
+      wget https://raw.githubusercontent.com/kubecost/cost-model/develop/kubernetes/exporter/exporter.yaml
+      ```
+
+    1.b.
+      On the line
+
+      ```
+      value: "{{prometheusEndpoint}}" # The endpoint should have the form http://<service-name>.<namespace-name>.svc
+      ```
+
+      of `exporter.yaml`, substitute your own Prometheus URI for `{{prometheusEndpoint}}`
+
+    1.c.
+
+      ```
+      kubectl apply -f exporter.yaml --namespace cost-model
+      ```
```

Comment:
Explaining that `{{prometheusEndpoint}}` has to be set in order for the `apply` to work